Lets Do This/Controllers/Settings/LDTSettingsViewController.m

```diff
@@ -163,8 +163,14 @@ - (void)handleLogoutTap:(UITapGestureRecognizer *)recognizer {
             LDTTabBarController *tabBar = (LDTTabBarController *)[[[[UIApplication sharedApplication] delegate] window] rootViewController];
             [tabBar setSelectedIndex:0];
         } errorHandler:^(NSError *error) {
+            // Performs normal logout functionality even if we are presented with an error.
             [SVProgressHUD dismiss];
             [LDTMessage displayErrorMessageForError:error];
+            [self.navigationController pushViewController:[[LDTUserConnectViewController alloc] init] animated:YES];
+            [self.navigationController styleNavigationBar:LDTNavigationBarStyleClear];
+            [LDTMessage setDefaultViewController:self.navigationController];
+            LDTTabBarController *tabBar = (LDTTabBarController *)[[[[UIApplication sharedApplication] delegate] window] rootViewController];
+            [tabBar setSelectedIndex:0];
```

Comment:
This code should be DRY to a new method like `presentUserConnectViewController` to be called both the completion and error handlers.